packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -114,22 +130,27 @@ export function makeUrqlClient (config: UrqlClientConfig): Client {
     makeCacheExchange(),
     ssrExchange({
       isClient: true,
-      initialState: window.__CYPRESS_INITIAL_DATA__ ?? {},
+      // @ts-ignore - this seems fine locally, but on CI tsc is failing - bizarre.
+      initialState: (window.__CYPRESS_INITIAL_DATA__ ? JSON.parse(decodeBase64Unicode(window.__CYPRESS_INITIAL_DATA__)) : {}),
     }),
     namedRouteExchange,
     // TODO(tim): add this when we want to use the socket as the GraphQL
     // transport layer for all operations
     // target === 'launchpad' ? fetchExchange : socketExchange(io),
     fetchExchange,
-  ]
+  )
 
   if (import.meta.env.DEV) {
     exchanges.unshift(devtoolsExchange)
   }
 
   return createClient({
     url: GRAPHQL_URL,
-    requestPolicy: 'cache-and-network',
+    // TODO(lachlan): we should use 'cache-only' in run mode for performance.
```

Comment:
I feel like this is no longer a TODO, since we're doing it. But the comment itself is still valid. Possibly you mentioned a TODO  around just not creating the client at all in run mode though, since we don't in theory need any gql requests at all.

Reply:
removed